scripts/host_cap.py

```diff
@@ -109,6 +109,14 @@ def constituent_num_consts_funcname(host_model):
     Because this is a user interface API function, the name is fixed."""
     return "{}_ccpp_number_constituents".format(host_model.name)
 
+###############################################################################
+def query_scheme_constituents_funcname(host_model):
+###############################################################################
+    """Return the name of the function to return True if the standard name
+    passed in matches an existing constituent
+    Because this is a user interface API function, the name is fixed."""
+    return "{}_ccpp_is_scheme_constituent".format(host_model.name)
```

Comment:
I think this can be an f-string (which will make pylint happy):

```suggestion
    return f"{host_model.name}_ccpp_is_scheme_constituent"
```

Reply:
changed to f-string